addons/mail/static/src/components/thread_viewer/thread_viewer_tests.js

```diff
@@ -476,6 +476,33 @@ QUnit.test('show message subject if thread is mailing channel', async function (
     )
 });
 
+QUnit.test('should not display user notification messages in non-mailboxes', async function (assert) {
```

Comment:
According to the fix and the comment that was added, test should be updated to be this? So it should be moved into chatter tests?
```suggestion
QUnit.test('should not display user notification messages in chatter', async function (assert) {
```